### PR TITLE
add text and vision dpv2 components, jobs and tests to azureml-assets

### PR DESCRIPTION
**Pull request by imatiach-msft**
Add text and vision dpv2 components, jobs and tests to azureml-assets.

This includes:
1.) The RAI Text Insights component and the DBPedia dataset for multiclass text classification
2.) The RAI Vision Insights component and the Fridge dataset for multiclass image classification

This is required to onboard to the azureml preview registry for RAI text and vision components.

### CODE REVIEW

File: responsibleai/vision/jobs/prepare.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+from azure.ai.ml.entities import AmlCompute, PipelineJob
+from azure.ai.ml import dsl, load_component, MLClient
+from azure.identity import DefaultAzureCredential
+from azureml.core import Workspace
```
Comment (kicha0) [inline responsibleai/vision/jobs/prepare.py:7]: I think azureml.core does not exist in the test environment, you could either find dpv2 equivalent or update the requirments.txt (can be done after merging this)

https://msdata.visualstudio.com/DefaultCollection/Vienna/_git/azureml-asset?path=/registry/requirements.txt

File: responsibleai/vision/components/rai_vision_insights/spec.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+$schema: http://azureml/sdk-2-0/CommandComponent.json
+name: rai_vision_insights
+display_name: RAI Vision Insights
+version: 2
```
Comment (kicha0) [inline responsibleai/vision/components/rai_vision_insights/spec.yaml:4]: for component in private / public preview, should we use semvar versioning and add preview at the end, for example 0.0.1.preview

**Review by kicha0 — COMMENTED** [review]
please update versioning of the component.

File: responsibleai/vision/components/rai_vision_insights/spec.yaml (hunk 2 above)
Comment (imatiach-msft) [reply responsibleai/vision/components/rai_vision_insights/spec.yaml:4]: done!

File: responsibleai/vision/jobs/prepare.py (hunk 1 above)
Comment (imatiach-msft) [reply responsibleai/vision/jobs/prepare.py:7]: I added some try/except logic, this should not be an issue in the builds

**Review by kicha0 — APPROVED** [review]

**Pull request merged**